src/doc/trpl/operators-and-overloading.md

```diff
@@ -1,3 +1,85 @@
 % Operators and Overloading
 
-Coming soon!
+Rust does not have general support for overloading operators or creating new
+operators. However, there are certain operators in Rust that _are_ able to
```

Comment:
I found these first two sentences a little confusing, it sounded like we don't have overloading but then it says we do? Perhaps 100% custom operators could be mentioned at the end of this section?